X/XLA/build_tarballs.jl

```diff
@@ -0,0 +1,45 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "XLA"
+version = v"0.0.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/openxla/xla.git", "3e0222a9a4f11b5f47ed6c3bcbbdaeee81e2bc68")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+apk update
+apk add bazel --repository=http://dl-cdn.alpinelinux.org/alpine/edge/testing
+apk add py-numpy
```

Comment:
`py3-numpy`: https://pkgs.alpinelinux.org/package/v3.15/community/x86_64/py3-numpy?
```suggestion
apk add py3-numpy
```